packages/schema/src/Schema.ts

```diff
@@ -4012,6 +4012,70 @@ export class Lowercased extends String$.pipe(
   static override annotations: (annotations: Annotations.Schema<string>) => typeof Lowercased = super.annotations
 }
 
+/**
+ * @category type id
+ * @since 0.67.0
+ */
+export const CapitalizedTypeId: unique symbol = Symbol.for("@effect/schema/TypeId/Capitalized")
+
+/**
+ * Verifies that a string is capitalized.
+ *
+ * @category string filters
+ * @since 0.67.0
```

Comment:
```suggestion
 * @since 0.68.18
```

(idem for the other since tags)